### ISSUE TITLE
Discriminator loss function

### ISSUE THREAD

**Issue by IvanBarabanau** [znxlwm/UGATIT-pytorch#50]
Could you, please, clarify, why MSE loss in discriminator compares unnormalized logits with 0/1 values?

**Comment by WangDingok** [comment]
see detail in lsgan paper
